<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>198</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLineEdit" name="downLink">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>30</y>
     <width>481</width>
     <height>25</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Enter Download Link</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="saveLoc">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>70</y>
     <width>481</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>Choose Save Location</string>
   </property>
  </widget>
  <widget class="QPushButton" name="checkButton">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>30</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="saveButton">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>70</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QProgressBar" name="progressBar">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>110</y>
     <width>601</width>
     <height>23</height>
    </rect>
   </property>
   <property name="value">
    <number>0</number>
   </property>
  </widget>
  <widget class="QPushButton" name="downloadButton">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>150</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Download</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
